src/components/KeplrConnection.tsx

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useState } from 'react';
+import { useAtom, useAtomValue, useSetAtom } from 'jotai';
+import { toast } from 'react-toastify';
+import { Oval } from 'react-loader-spinner';
+import {
+  makeAgoricKeplrConnection,
+  AgoricKeplrConnectionErrors as Errors,
+} from '@agoric/web-components';
+import 'react-toastify/dist/ReactToastify.css';
+
+import {
+  brandToInfoAtom,
+  pursesAtom,
+  offersAtom,
+  instanceIdsAtom,
+  governedParamsIndexAtom,
+  metricsIndexAtom,
+  keplrConnectionAtom,
+  networkConfigAtom,
+} from 'store/app';
+import { watchContract, watchPurses } from 'utils/updates';
+import NetworkDropdown from 'components/NetworkDropdown';
+
+import 'styles/globals.css';
+import clsx from 'clsx';
+
+const KeplrConnection = () => {
```

Comment:
this isn't really about Keplr. Keplr is for signing. This should work without Keplr if given an address and network config. consider `NetworkConnection` or `NetworkConfigurator`.

the only thing dapp-psm should use Keplr for is to get the address.

Reply:
Renamed to `ChainConnection`